docs/src/content/components/icon/propData.js

```diff
@@ -4,6 +4,12 @@ const propData = [
   ['name', 'Name of icon that matches material names', 'string', ''],
   ['size', 'Size of icon', 'number', ''],
   ['style', 'Styles root element', 'object', ''],
+  [
+    'iconComponent',
```

Comment:
I don't think these are being sorted alphabetically automatically right now, so I think it needs to be manually put there or write a function to automatically sort these props